Update issue templates (#2209)
microsoft/BotFramework-WebChat: .github/ISSUE_TEMPLATE/feature_request.md

title: ''
labels: Enhancement, Pending
assignees: ''

---

<!--
To let use better understand your request, please follow this template and fill out as comprehensive as possible.

> Creative ideas are not uncommon. Please check this query to see if anyone already on the same topic

Reviewer: ```suggestion
Creative ideas are not uncommon. Please check the following to see if anyone has already requested the same feature:
```

https://github.com/microsoft/botframework-Webchat/issues?q=is%3Aissue+is%3Aopen+label%3ASample
-->

## User story

> Today, auto complete is not implemented in the send box.

Reviewer: Again, this should be in comments, not prefilled for the user.

Tomorrow, I want to add auto complete to the send box. Our users are mostly on mobile devices. Tapping buttons than typing make the chatbot experience more attractive.

> <!-- Please attach sketches with annotations, if any -->

Reviewer: sketches?

## Alternatives

> I have tried suggested actions but it does not fit in the user story because:

Reviewer: this should be commented

- They are not dynamic
- Only 5 of them is shown

## Other implementations

> App ABC and XYZ implemented this feature. In app ABC, they show the auto-completes using inlined text. In app XYZ, they show the auto-completes on top of the input box.

Reviewer: Should be commented.
